client/components/common/Meta/BaseSelect.vue

```diff
@@ -1,92 +1,50 @@
 <template>
-  <div
-    :class="{ active }"
-    class="select">
-    <label :for="meta.key">{{ meta.label }}</label>
-    <multiselect
-      @open="active = true"
-      @close="active = false"
-      @input="update"
-      :value="value"
-      :name="meta.key"
-      :options="options"
-      :searchable="false"
-      :placeholder="meta.placeholder"
-      :multiple="isMultiSelect">
-      <template slot="option" slot-scope="{ option }">
-        <img v-if="option.img" :src="option.img" :alt="option.label" class="img">
-        <span>{{ option.label }}</span>
-      </template>
-      <template slot="singleLabel" slot-scope="{ option }">
```

Comment:
You need to handle img options as-well 😉Please sync with @mcapeta if you have any additional questions.

Reply:
My bad 🙈 moved it back in:
![Screen Shot 2019-11-29 at 11 02 23](https://user-images.githubusercontent.com/20609558/69861199-3020b400-1298-11ea-8917-6b67e2e252e8.png)